addons/website/static/src/scss/website.scss

```
@@ -732,158 +732,116 @@ table.table_desc tr td {
 .s_carousel,
 .s_quotes_carousel {
 
+    --o-carousel-indicators-dots-size: .75rem;
+    --o-carousel-controllers-size: #{map-get($spacers, 5)};
+
     // Controls
-    .carousel-control-prev,
-    .carousel-control-next {
-        position: absolute;
-        cursor: pointer;
-        width: 8%;
-        opacity: 1;
-    }
     @include media-breakpoint-down(md) {
         .carousel-control-prev,
         .carousel-control-next {
             display: none; // remove arrows on mobile
         }
     }
-    .carousel-control-prev { justify-content: flex-start; }
-    .carousel-control-next { justify-content: flex-end; }
-    .carousel-control-prev-icon,
-    .carousel-control-next-icon {
-        width: auto;
-        height: auto;
-        background-image: none;
-        color: $body-color;
-        &:before {
-            font-family: "FontAwesome";
-            display: inline-block;
-            background-color: #fff;
+
+    &.carousel-dark {
+        --o-carousel-indicator-dots-bg-color: #{to-rgb(#fff)};
+    }
+
+    .carousel-control-prev,
+    .carousel-control-next {
+        position: absolute;
+        width: 10%;
+    }
+
+    &.s_carousel_default {
+        .carousel-control-prev, .carousel-control-next {
+            &:hover {
+                background: rgba($carousel-dark-indicator-active-bg, .2);
+            }
+        }
+        &.carousel-dark {
```

Convention: empty line between selectors 😉
```suggestion
        }
       
        &.carousel-dark {
```